lib/Doctrine/Import/Builder.php

```diff
@@ -783,7 +783,7 @@ public function buildPhpDocs(array $definition)
             if (isset($definition['relations']) && ! empty($definition['relations'])) {
                 foreach ($definition['relations'] as $relation) {
                     $type = (isset($relation['type']) && $relation['type'] == Doctrine_Relation::MANY) ? 'Doctrine_Collection' : $this->_classPrefix . $relation['class'];
-                    if ((isset($relation['type']) ? $relation['type'] : null) == Doctrine_Relation::ONE) {
+                    if ($relation["type"] == Doctrine_Relation::ONE) {
```

Comment:
This would raise a warning, if type is not set in `$relation`, doesn't it?

Reply:
reverted, this might be a change wrongly introduced by a rebasing